### ISSUE TITLE
Read Shape files from a standard directory and read by name

### ISSUE THREAD

**Issue by zikani03** [sethbonnie/dblstd#6]
Currently shape files must be passed a path to the file. 

The program should allow specifying just the name of the Shape file which should be read from a standard directory. I suggest `$HOME/.config/dblstd` or `$HOME/.dblstd/`

Now let us assume we have the following files `$HOME/.config/dblstd/go.shape` and `$HOME/.config/dblstd/java.shape`

This should then allow the user to specify just the name of the shape file 

```sh
$ dblstd -s go /path/to/my/go/project
$ dblstd -s java /path/to/java/project
```

This should not conflict with a shape file in the PWD .e.g `go.shape` - as the user will have to specify the full name of that file i.e. 
```sh
$ dblstd -s go.shape /path/to/my/go/project
```

**Comment by sethbonnie** [comment]
While I like the idea, I'm not a big fan of conditionals. When you overload what `-s` does, you now need to parse the given string to make sure it doesn't have the `.shape` extension. Meaning in the future if there was anything extra you wanted to do for this flag, you now have to consider 2 branches instead of one.

Why not just use another flag to use a standard shape from the config directory?

```sh
# -c (-config) specifies to use a shape file from the dblstd config directory
$ dblstd -c go /path/to/my/go/project
$ dblstd -c java /path/to/java/project
```

I'd imagine we'd also want to read from an env var in case we want to have another directory (other than the default shape directory). I'm thinking something like:

```sh
$ SHAPE_DIR=$HOME/alt/shape/dir dblstd -c go /path/to/my/go/project
```

**Comment by zikani03** [comment]
I understand your concern and think using another command-line flag is okay. I suppose the program will just have to decide what happens in a situation where one uses both flags:

```sh
$ dblstd -c go -s ./java.shape /path/to/my/go/project
```
As for the actual flag name, I would suggest maybe using `-u` since we may want to use `-c --config` for global configuration, (unless we don't see the need to have a config file that's != shape file) :

```sh
$ dblstd -c dblstd.yaml -u go /path/to/my/go/project
```

I like the idea to support overriding the directory with an env var 👍🏾

**Comment by sethbonnie** [comment]
Hmm, how about

`-s` for shape
`-f` for explicit file

Defaults:
```sh
# looks for go.shape in current directory
# if not found, looks for go.shape in default config directory (default value of $DBLSTD_CONFIG -- $HOME/.config/dblstd)
# if not found in config dir, error out
$ dblstd -s go /path/to/go/project
```

Overrides:
```sh
# override shape lookup path with explicit file via -f
$ dblstd -s go -f ./path/to/shapes/go.shape /path/to/go/project
```

**Comment by zikani03** [comment]
Yeah, I thought about using `-f` as well. I could live with that; my use cases would probably be more of reading the file from the configured directory anyways :)
